ir/type.def

```diff
@@ -263,6 +263,17 @@ abstract Type_StructLike : Type_Declaration, INestedNamespace, ISimpleNamespace,
         }
         return -1;
     }
+    // This function returns offset in bits.
```

Comment:
Please use a /// comment to describe what this function does. You also need to document the -1 case.
You should also mention what this does when fields do not have a defined width (e.g., enums or varbits).